tests/phpunit/tests/term.php

```diff
@@ -308,4 +308,52 @@ public function test_orphan_category() {
 		$cat_id2 = self::factory()->category->create( array( 'parent' => $cat_id1 ) );
 		$this->assertWPError( $cat_id2 );
 	}
+
+
+	/**
+	 * @ticket 58329
+	 *
+	 * @dataProvider data_get_term_filter
+	 *
+	 * @param string $filter Term to filter.
+	 */
+	public function test_get_term_filter( $filter ) {
+		$cat_id1 = self::factory()->category->create();
+
+		$term = get_term( $cat_id1, '', OBJECT, $filter );
+
+		$this->assertSame( $filter, $term->filter, 'Sanitize should match' );
```

Comment:
```suggestion
		$this->assertSame( $filter, $term->filter, "The term's 'filter' property should be set to '$filter'." );
```
Some added clarity if the assertion fails without increasing the maintenance burden.

![image](https://github.com/WordPress/wordpress-develop/assets/79332690/6c06cb2c-6010-43d1-a6ef-997fc7ae7451)

with `--testdox`:

![image](https://github.com/WordPress/wordpress-develop/assets/79332690/0423ba34-cbee-4f94-a0e0-571e2d23554d)